hack/lint-gofmt.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+WMCO_ROOT=$(dirname "${BASH_SOURCE}")/..
+
+GO_VERSION=($(go version))
+
+if [[ -z $(echo "${GO_VERSION[2]}" | grep -E 'go1.13') ]]; then
```

Comment:
Aren't we still on go 1.12  in our CI builds?

Reply:
When I ran `make new-repo` in the release repo it defaulted to 1.13, so I am assuming that they have moved to 1.13. Moreover, 1.13 a requirement if we are using `operator-sdk`.

Reply:
>  so I am assuming that they have moved to 1.13. Moreover, 1.13 a requirement if we are using operator-sdk.

That's correct. Let's see if this passes the tests.